site/monthly-updates-feed.njk

```diff
@@ -0,0 +1,82 @@
+---json
+{
+  "permalink": "/release-notes.xml",
+  "eleventyExcludeFromCollections": true,
+  "metadata": {
+    "title": "Web Platform Monthly Updates",
+    "subtitle": "Monthly updates on the web platform features.",
+    "language": "en",
+    "url": "https://web-platform-dx.github.io/web-features-explorer/",
+    "author": {
+      "name": "Web Platform Team"
+    }
+  }
+}
+---
+<?xml version="1.0" encoding="utf-8"?>
+<feed xmlns="http://www.w3.org/2005/Atom" xml:base="{{ metadata.url }}">
+  <title>{{ metadata.title }}</title>
+  <subtitle>{{ metadata.subtitle }}</subtitle>
+  <link href="https://web-platform-dx.github.io/web-features-explorer/release-notes.xml" rel="self"/>
+  <link href="{{ metadata.url }}"/>
+  <updated>{{ collections.perMonth | getNewestCollectionItemDate | dateToRfc3339 }}</updated>
+  <id>{{ metadata.url }}</id>
+  <author>
+    <name>{{ metadata.author.name }}</name>
+  </author>
+  {%- for month in perMonth %}
```

Comment:
I _think_ we may want to ignore the first month in the list. So, right now, this would be September 2024. And the reason for this is that features are still being added to that month's release note as time goes by. Only when the month is over do we have the final list of features.

For example, imagine browser X releases its next version N on September 30. Now imagine browser X is the last browser one to implement feature A, and support is coming in version N. Right now (on September 16), the feature is not in the September 2024 release notes page. But, after September 30, it will be.

What do you think?
We could also change the logic in .eleventy.js instead, and add the features who _will_ be baseline in each month, even if that month hasn't completely passed yet.

Reply:
Or we just let the current entry update itself as time goes by, which is also another option. I just don't know how RSS readers deal with updated entries.

Reply:
If the `<updated>` date of the FIRST MONTH item is also updated, then including FIRST MONTH may not be a problem. (In the current implementation, the `<updated>` date is fixed, so this may be problematic.)
However, many RSS readers use `<id>{{ metadata.url }}</id>` to determine uniqueness, so there is no guarantee that once an item is marked as read, it will be displayed again even if the update date is updated.

To be safe, it is appropriate not to include first month items in RSS.

Reply:
fixed in https://github.com/web-platform-dx/web-features-explorer/pull/23/commits/ceef1df18c9f8d590ba73705b525dda75847b787